<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mamad's Terminal</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>KDE_Logo_compress.png</normaloff>KDE_Logo_compress.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="txtPort">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>COM1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Device Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Parameters : 9600-8-n-1</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtData">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>241</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBuffer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>241</width>
      <height>23</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILE">
    <property name="title">
     <string>FILE</string>
    </property>
    <addaction name="actionClear"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuCOMM">
    <property name="title">
     <string>COMM</string>
    </property>
    <widget class="QMenu" name="menuConnect">
     <property name="title">
      <string>Connect</string>
     </property>
     <addaction name="actionOpen"/>
     <addaction name="actionClose"/>
    </widget>
    <addaction name="menuConnect"/>
   </widget>
   <addaction name="menuFILE"/>
   <addaction name="menuCOMM"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionClose"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
